modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -81,6 +81,11 @@ class Create_Timepoint extends \NDB_Form
         );
 
         $this->tpl_data['success'] = true;
+        $this->tpl_data['candID']  = htmlentities(
```

Comment:
why are we doing this twice, once in process() and once in setup() ?

Reply:
You're right it only needs to happen in setup. I'll change this in a bit.

Reply:
Actually looking at this again it looks like we need to do it twice... In both functions an element is added to the front-end and it needs to be sanitized there. 

From what I know of LORIS it seems like some functionality is duplicated here and I'm not sure that `_process` should be creating HTML elements. It should probably be refactored but that's not the goal of this PR so the sanitization should occur in both place.

Reply:
but i believe that you added the addition of the element to the front-end in process(). like 84 to add the identifier to tpl_data wasn't there before. what was the reason for adding it?

Reply:
Excellent point... Sorry I was working on another PR when looking at this and was distracted. I'll remove it.